markets/spot-market/README.md

```diff
@@ -38,39 +38,46 @@ Traders may simulate calling these functions with `callStatic` to retrieve a quo
 
 ### Asyncronous Orders
 
-Asyncronous orders involve two transactions: a _commitment_ and a _settlement_. This reduces composability, but allows for front-running mitigation such that lower fees can be offered to traders. Asyncronous orders may be cancelled under certain circumstances. The relevant logic is included in the [Async Order Module](./contracts/modules/AsyncOrderModule.sol).
+Asyncronous orders involve two transactions: a _commitment_ and a _settlement_. This reduces composability, but allows for front-running mitigation such that lower fees can be offered to traders via `FeeConfiguration.asyncFixedFee`. Asynchronous orders may only be cancelled prior to settlement if it is
+outside of the settlement window.
```

Comment:
Fix line break and add "The relevant logic is included in the [Async Order Module](./contracts/modules/AsyncOrderModule.sol)."

Reply:
not "prior to settlement" exactly?